beep/featurize.py

```diff
@@ -417,6 +421,87 @@ def process_file_list_from_json(file_list_json, processed_dir='data-share/featur
     return json.dumps(output_data)
 
 
+def get_cycle_life(processed_cycler_data, n_cycles_cutoff=40, threshold=0.8):
```

Comment:
For completeness sake, would it be possible to include unit tests for these? I suppose you can re-purpose the ones from `test_structure.py`, since the functionality shouldn't have changed.